doc/docs.md

```diff
@@ -18,15 +18,15 @@ you can do in V.
 <ol>
 <table>
 	<tr>
-		<td><a href='#hello-world'>1. Hello world</a></td>
-		<td><a href='#comments'>2. Comments</a></td>
-		<td><a href='#functions'>3. Functions</a></td>
-		<td><a href='#constants--variables'>4. Variables</a></td>
-		<td><a href='#primitive-types'>5. Primitive types</a></td>
-		<td><a href='#strings'>6. Strings</a></td>
-		</tr>
+		<td><a href='#hello-world'>Hello world</a></td>
```

Comment:
Also... I don't think we should include a number

Reply:
They are needed so that it's clear it goes from left to right

Reply:
Why don't use lists instead?

Reply:
The don't work inside tables, the layout is off